server/tests/feeds/feeds.ts

```diff
@@ -99,6 +99,13 @@ describe('Test syndication feeds', () => {
       await servers[0].comments.createThread({ videoId: id, text: 'comment on unlisted video' })
     }
 
+    {
+      const attributes = { name: 'password protected video', privacy: VideoPrivacy.PASSWORD_PROTECTED, videoPasswords: [ 'password' ] }
```

Comment:
Do you also check we don't return password protected videos in the videos feeds?

Reply:
This is implicitly tested here as it is for the unlisted video: https://github.com/Chocobozzz/PeerTube/blob/9016d9ec3da935d6596453dd855ac2ca7af3a5a1/server/tests/feeds/feeds.ts#L312